src/types.d.ts

```diff
@@ -1,15 +1,15 @@
 import { type FunctionDefinition, type FunctionDefinitionImage } from 'serverless';
 
-type ServerlessFunctions = { [key: string]: FunctionDefinition | FunctionDefinitionImage };
+export type ServerlessFunctions = { [key: string]: FunctionDefinition | FunctionDefinitionImage };
 
-type FunctionName = string;
+export type FunctionName = string;
 
-type InternalPluginConfig = {
+export type InternalPluginConfig = {
   enabled: boolean;
   enabledFunctions: FunctionName[];
 };
 
-type ServerlessPluginConfig = {
+export type ServerlessPluginConfig = {
```

Comment:
Is it better to have explicit type export?

Reply:
they were imported in some other files which means they needed to be exported